### DISCUSSION TITLE
Zabbix, multiple db backup containers

### DISCUSSION THREAD

**Discussion by june012006** [tiredofit/docker-db-backup#356] [Q&A] [answered]
I'm new to using zabbix, so forgive me if the answer is obvious, but can multiple db backups be monitored from the same template and docker host?    I see in the template that it's named dbbackup4, and I'm wondering how it's differentiated.

**Comment by tiredofit** [comment]
The template is built to support multiple instances within the same container using Discovery features.
After a backup in your latest data you should see information populate..

**Reply by june012006** [reply]
What about multiple containers on a docker host?   For example, I'll be backing up authentik's postgres and phpipam's mariadb, as well as various other postgres dbs.   I gather already that I'll need an instance per db or stack, but how is that tracked in zabbix?

**Reply by tiredofit** [reply, marked as answer]
If you are backing up multiple hosts via the same DB Backup container it will track it.

eg 

```
DB01_HOST=authentik-app
DB01_NAME=authentik-db
DB02_HOST=randompostgres
DB02_NAME=postgresdb_name 
``` 

Both will get tracked and metrics sent to zabbix server upon backup initiation.
Recently before the 4.xx series one would have needed multiple containers for db-backup. I was recommending one per stack. In fact I still recommend it for portability purposes.

**Reply by june012006** [reply]
Ok, that's what I was looking for, thanks.     Is there any special setup for the zabbix end?

**Reply by tiredofit** [reply]
Not really. Server or proxy should be available to access from the container as any stats from DB-Backup'd are pushed as opposed to pulled..

**Reply by june012006** [reply]
Do I add the template as a host?    Sorry, but I am really new to zabbix, so the meriad of settings are a little overwhelming atm
